frontend/public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="An interactive playground for Playwright with various examples."/>
+    <meta name="description" content="An interactive Playground for Playwright with various examples available."/>
```

Comment:
Probably it's more appropriate for playground to be lowercase here since it's not a proper noun.